src/globalNotification/globalNotification.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {
+  IconAlertCircleFill,
+  IconCheckCircleFill,
+  IconErrorFill,
+  IconInfoCircleFill,
+  Notification,
+  NotificationType,
+} from 'hds-react';
+import { useTranslation } from 'react-i18next';
+import { useGlobalNotifications } from './globalNotificationProvider';
+
+export interface Props {
+  id: string;
+  label?: string;
+  icon?: boolean;
+  body: string;
+  type: NotificationType;
+}
+
+const GlobalNotification = (props: Props): JSX.Element | null => {
+  const { t } = useTranslation();
+  const { popNotification } = useGlobalNotifications();
+
+  const getIcon = (type: NotificationType): JSX.Element => {
+    switch (type) {
+      case 'alert':
+        return (
+          <IconAlertCircleFill className="GlobalNotificationContainer__notification-icon--alert" />
```

Comment:
Don't forget to also add the class name the modifier modifies (i.e. `GlobalNotificationContainer__notification-icon`.

I would personally try to make this a bit less repetitive and more robust by only assigning the component type into a variable and then using it like so:

```tsx
import { IconProps /* ... */ } from 'hds-react';

let IconComponent: React.FC<IconProps>;
switch (type) {
    case 'alert':
        IconComponent = IconAlertCircleFill;
        break;
    /* etc. */
}

return <IconComponent className={classNames(
    'GlobalNotificationContainer__notification-icon',
    `GlobalNotificationContainer__notification-icon--${type}`)}
/>;
```

The types could even be specified as an enum, I think.